manifests/config.pp

```diff
@@ -40,15 +40,7 @@
     }
   }
 
-  $_config.each |$setting, $value| {
-    ini_setting { "${config_file} ${setting} ${value}":
-      ensure  => present,
-      path    => $config_file,
-      section => '',
-      setting => $setting,
-      value   => $value,
-      require => File['/etc/letsencrypt'],
-    }
-  }
+  $_config_joined = join_keys_to_values($_config, '=')
+  letsencrypt::config::ini { $_config_joined: }
```

Comment:
This seems to fail on Puppet 4.0 and 4.1, but not on 4.2 or 3.x, with a duplicate declaration error:

    Evaluation Error: Error while evaluating a Function Call, Duplicate declaration: Ini_setting[[] email] is already declared; cannot redeclare at /home/travis/build/domcleal/puppet-letsencrypt/spec/fixtures/modules/letsencrypt/manifests/config.pp:48:3

I can't see any reason for it, and removing the create_resources() call inside the function shows that the ini_settings resource isn't there from somewhere else.  I suppose it's a Puppet bug, but not one I'm aware of.